lib/use-checkout-pricing.js

```diff
@@ -38,20 +47,39 @@ export default function useCheckoutPricing(initialInputs = {}, handleError = thr
   const recurly = useRecurly();
   const [loading, setLoading] = useState(false);
   const [input, setInput] = useState(initialInputs);
-  let checkoutPricing = recurly.Pricing.Checkout();
-  const [pricing, setPricing] = useState(checkoutPricing);
+  const [pricing, setPricing] = useState(recurly.Pricing.Checkout());
 
   useEffect(() => {
     setLoading(true);
-    const { subscriptions = [] } = input;
+    let checkoutPricing = recurly.Pricing.Checkout();
+    const { subscriptions = [], ...singleInputs } = input;
+
+    checkoutPricing = addSingleInputs(singleInputs, checkoutPricing);
```

Comment:
nit: Thinking `singleInputs` could use a better name. `rest` does seem appropriate, or perhaps `scalarInputs`.

Reply:
I've been thinking this the entire time as well. I picked this var name like 20 seconds into writing the feature thinking I'd think of something later and go back and update and never did. Definitely needs a better name.

Reply:
https://github.com/recurly/react-recurly/pull/25/commits